docs/docs/developers/sandbox/references/sandbox-reference.md

```diff
@@ -61,35 +61,25 @@ If you wish to run components of the Aztec network stack separately, you can use
 aztec start --node [nodeOptions] --pxe [pxeOptions] --archiver [archiverOptions] --sequencer [sequencerOptions] --prover [proverOptions] ----p2p-bootstrap [p2pOptions]
 ```
 
-Starting the aztec node alongside a PXE, sequencer or archiver, will attach the components to the node. If you want to e.g. run a PXE separately to a node, you can:
-Start a node:
-
-```bash
-aztec start --node [node] --archiver [archiverOptions]
-```
-
-Then start a PXE on a separate terminal that connects to that node:
-
-```bash
-aztec start --pxe nodeUrl=http://localhost:8080
-```
+Starting the aztec node alongside a PXE, sequencer or archiver, will attach the components to the node.Eg if you want to run a PXE separately to a node, you can [read this guide](../guides/run_more_than_one_pxe_sandbox.md)/
 
 ## Environment Variables
 
 There are various environment variables you can use when running the whole sandbox or when running on of the available modes.
 
+To change them, you can open `~/.aztec/docker-compose.yml` and edit them directly.
+
 **Sandbox**
 
 ```sh
 DEBUG=aztec:* # The level of debugging logs to be displayed. using "aztec:*" will log everything.
-ETHEREUM_HOST=http://ethereum:8545 # The Ethereum JSON RPC URL. We use an anvil instance that runs in parallel to the sandbox on docker by default.
 HOST_WORKDIR='${PWD}' # The location to store log outpus. Will use ~/.aztec where the docker-compose.yml file is stored by default.
+ETHEREUM_HOST=http://ethereum:8545 # The Ethereum JSON RPC URL. We use an anvil instance that runs in parallel to the sandbox on docker by default.
 CHAIN_ID=31337 # The Chain ID that the Ethereum host is using.
 TEST_ACCOUNTS='true' # Option to deploy 3 test account when sandbox starts. (default: true)
-DEPLOY_AZTEC_CONTRACTS='true' # Option to deploy the Aztec contracts when sandbox starts. (default: true)
 MODE='sandbox' # Option to start the sandbox or a standalone part of the system. (default: sandbox)
-AZTEC_NODE_PORT=8079 # The port that the Aztec node wil be listening to (default: 8079)
 PXE_PORT=8080 # The port that the PXE will be listening to (default: 8080)
+SANDBOX_PORT=8080 # The port that Aztec Sandbox wil be listening to (default: 8080)
```

Comment:
Are `PXE_PORT` and `SANDBOX_PORT` the same thing? Or is `SANDBOX_PORT` replacing `AZTEC_NODE_PORT`?

Reply:
`PXE_PORT` is a different thing - if you just run the sandbox theyre run on the same port, but you can run the sandbox and another PXE separately

`AZTEC_NODE_PORT` env var was removed a while back for some reason, but youre right in that this naming makes more sense so Ive changed `SANDBOX_PORT` to `AZTEC_NODE_PORT`